ocp_resources/datavolume.py

```diff
@@ -311,6 +314,7 @@ def test_dv():
                 if sample and sample.get("status", {}).get("phase") == self.Status.SUCCEEDED:
                     break
                 elif sample is None and dv_garbage_collection_enabled:
+                    warn(gc_deprecatio_warning)
```

Comment:
deprecation should be added to `dv_garbage_collection_enabled` in docstring and at the beginning of the function:
``` 
    if dv_garbage_collection_enabled is not None:
        warn....
```